apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/RifLayoutsProcessor.java

```diff
@@ -707,6 +982,47 @@ private TypeSpec generateHeaderEntity(MappingSpec mappingSpec) throws IOExceptio
       headerEntityClass.addMethod(childGetter);
     }
 
+    // Add the parent-to-child join field and accessor, if appropriate.
+    if (mappingSpec.getHasEnrollments()) {
+
+      ParameterizedTypeName childFieldType =
+          ParameterizedTypeName.get(ClassName.get(List.class), mappingSpec.getEnrollmentEntity());
+
+      FieldSpec.Builder childField =
+          FieldSpec.builder(childFieldType, "enrollments", Modifier.PRIVATE)
+              .initializer("new $T<>()", LinkedList.class);
+
+      childField.addAnnotation(
+          AnnotationSpec.builder(OneToMany.class)
+              .addMember("mappedBy", "$S", mappingSpec.getEnrollmentEntityParentField())
+              .addMember("orphanRemoval", "$L", true)
+              .addMember("fetch", "$T.LAZY", FetchType.class)
```

Comment:
Not sure we want this to be lazy-loaded. Any thoughts?

Reply:
Eager loaded